src/protocol/request_response/mod.rs

```diff
@@ -63,9 +63,9 @@ mod handle;
 #[cfg(test)]
 mod tests;
 
-// TODO: add ability to specify limit for inbound requests?
-// TODO: convert inbound/outbound substreams to use `oneshot:Sender<()>` for sending/rejecting
-// responses.       this way, the code dealing with rejecting/responding doesn't have to block.
+// TODO: https://github.com/paritytech/litep2p/issues/339 Convert inbound/outbound
+// substreams to use `oneshot:Sender<()>` for sending/rejecting responses.
+// This way, the code dealing with rejecting/responding doesn't have to block.
```

Comment:
I think this is a stale comment because rejection/responding already uses a oneshot channel: https://github.com/paritytech/litep2p/blob/e0b7118ea2ca3e9f9beb9605f649d7e9dcdd24c4/src/protocol/request_response/handle.rs#L135